includes/class-user-update-watcher.php

```diff
@@ -0,0 +1,172 @@
+<?php
+/**
+ * Newspack Network watcher for updates on user' information
```

Comment:
WOW, this is a major issue! Not only the events are processed in the wrong order, but this can result in Nodes skipping events, because we are always querying for the most recent ones!

I committed a fix and we should get this published asap!

thanks for catching this @adekbadek !